### PR TITLE
Added seller company name in the body of 2nd reminder

### PR DESCRIPTION
**Pull request by ViggoWidoff**
Fixes from customer feedback

### CODE REVIEW

**Comment by ViggoWidoff** [comment]
@theilgaard Can you approve and merge asap?

File: email-templates/ALL/EN/DUNNING_SECOND_REMINDER.mjml [hunk 1]
```diff
@@ -41,7 +41,7 @@
   					    .link-nostyle { color: #0000FF; text-decoration: none }
  						   	</mj-style>
  								<!--Preview text-->      
- 								<mj-preview>We still haven't recieved your payment for {{ invoice_number }} by {{ company_name }}</mj-preview>
+ 								<mj-preview>We still haven't recieved your payment for inovice no. {{ invoice_number }} from {{ company_name }}</mj-preview>
```
Comment (theilgaard) [inline email-templates/ALL/EN/DUNNING_SECOND_REMINDER.mjml:44]: inovice

**Review by theilgaard — CHANGES_REQUESTED** [review]
Please proof read.

**Comment by ViggoWidoff** [comment]
Sorry @theilgaard It's fixed now

**Review by theilgaard — APPROVED** [review]

**Pull request merged**